recipes/gembs/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "conda_v2" %}
+{% set sha256 = "1ead3f20a36a40c0737ec58c2e783b9c1cb6a9de0317c0009964d1b164c598a0" %}
+{% set bcftoolsversion = "1.8" %}
+{% set bcftoolssha256 = "4acbfd691f137742e0be63d09f516434f0faf617a5c60f466140e0677915fced"%}
+
+package:
+  name: gembs
+  version: {{ version }}
+
+source:
+  - url: https://github.com/karl616/gemBS/archive/{{ version }}.tar.gz
```

Comment:
is this a fork from the original repo ? is the original one not being maintained anymore ?

Reply:
This is leftovers from testing, connected to the funky version above. Should be changed.